.eslintrc.yaml

```diff
@@ -4,10 +4,10 @@ plugins:
 
 rules:
   indent: [1, 2, {SwitchCase: 1}]
-  mozilla/balanced-listeners: [1]
   mozilla/no-aArgs: [1]
   mozilla/this-top-level-scope: [1]
   no-console: [1]
+  mozilla/balanced-listeners: [0]
```

Comment:
We'll probably want to not include this, once we get all the cleanup stuff added, like you were working on in #18.